gammapy/modeling/models/tests/test_cube.py

```diff
@@ -659,6 +659,13 @@ def test_fermi_isotropic():
     assert flux.unit == "MeV-1 cm-2 s-1 sr-1"
     assert isinstance(model.spectral_model, CompoundSpectralModel)
 
+    model_options = create_fermi_isotropic_diffuse_model(
+        filename=filename,
+        interp_kwargs={"extrapolate": True, "method": "nearest"},
+    )
+    flux_options = model_options(**coords)
+    assert_allclose(flux_options.value, 1.456e-13, rtol=1e-3)
```

Comment:
Can we test also an other extrapolation algorithm? If something changes in astropy, we will be then informed...

Reply:
Is 50 GeV outside the interpolation range? It would be good to test with an arrays of energies covering values inside and outside the interpolation range, such that the `interp_kwargs` make a difference

Reply:
It doesn't seem possible to have extrapolate False with this specific example, even if I take the energy directly from the file:

```
vals = np.loadtxt(make_path(filename))
energy = u.Quantity(vals[:, 0], "MeV")
coords_inner = {"lon": 0 * u.deg, "lat": 0 * u.deg, "energy": energy[7]}
flux = model(**coords_inner)
```
[interp_kwargs](https://github.com/user-attachments/assets/fa6a09a8-320f-44b0-b00c-a91e7d4bf5e8)

Reply:
Sorry the error actually comes from the 
`model = create_fermi_isotropic_diffuse_model(filename=filename,  interp_kwargs={"extrapolate": False})`